### DISCUSSION TITLE
Merge all names in one file

### DISCUSSION THREAD

**Discussion by ignFrau** [LabyMod/og-names#21] [Ideas]
Hey there 👋,

I have seen that several times now that the names don't work as they should.

Therefore, it would be better that we merge all names into one single txt file.

That means: We have a general txt file, where all names are inside and not country specific.

Let me know what you think. 

Have a nice day. ❤️

**Comment by 84** [comment]
Seems like it would be much harder to organize if it was.

**Comment by arktn** [comment]
I agree with @84, but I would suggest that we would sort the names according to their first letter. This would make it much easier to navigate the files and see if a name has already been listed. I also do not understand why you would sort them by their country it is destined to create duplicates since names can be popular in more than one country or even be from more than one country.

**Comment by ignFrau** [comment]
Thats what I mean, sorting them by country is a little bit too much effort.

So I would suggest a single file just with the names in it.

**Reply by arktn** [reply]
I understood your idea, but don't you think that having a file with a couple hundret lines is kind of hard to navigate?

**Reply by ignFrau** [reply]
I think it would be better yes.

I also think that you have a better overview, instead of having all the names spread over several files.

**Reply by arktn** [reply]
You got a point there, you definitely have a better overview over everything but navigating it would be a bit messy for most first time users. Nevertheless, having one big file or sorting it by letters is still a better way than sorting it by "their" country.

**Reply by ignFrau** [reply]
Agreed!

**Reply by arktn** [reply]
I tought about this again and I agree, one file would probably be the best option, we should go with the KISS principle here.

**Reply by ignFrau** [reply]
Good Idea! 👯‍♂️

**Comment by niklaswa** [comment]
We separated them to be able to use the top 1000 most popular names of each country

**Reply by ignFrau** [reply]
Okay Understood,

if you do it by each country it becomes very complicated and incorrect.

What if a person makes a pull request and adds all African names, then it becomes very very hard to find "OG" names...

That's why I meant to insert all names together and follow the Kiss principle _(Keep It Simple and Stupid)_ like @orsou mentioned.
